### PR TITLE
feat(boolean): add randomBool

### PR DESCRIPTION
**Pull request by AmirHosseinKarimi**

### CODE REVIEW

File: src/boolean/randomBool.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { randomInt } from '../number';
```
Comment (ASafaeirad) [inline src/boolean/randomBool.ts:1]: The file extension is required for the ESM support.

```suggestion
import { randomInt } from '../number.ts';
```

File: src/boolean/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { randomBool } from './randomBool';
```
Comment (ASafaeirad) [inline src/boolean/index.ts:1]:
```suggestion
export { randomBool } from './randomBool.ts';
```

**Review by ASafaeirad — CHANGES_REQUESTED** [review]

**Comment by ASafaeirad** [comment]
Hey mate, Thanks a lot for your contribution. :raised_hands:

**Comment by AmirHosseinKarimi** [comment]
Hey bro, Thank you for developing this awesome tool. :fire:

**Pull request merged**